### DISCUSSION TITLE
Deterministic way to run Eliza

### DISCUSSION THREAD

**Discussion by AgustinRamiroDiaz** [elizaOS/eliza#942] [Q&A]
Thank you very much for this project

I would like to use a combination of plugins available in the https://github.com/ai16z/eliza repository to create a script for a cron job. Specifically, I want to perform a deterministic operation, such as bridging SOL (Solana) to ETH (Ethereum). 

Is there a way to run Eliza with a deterministic script while bypassing its AI-based interactive components?

**Comment by odilitime** [comment]
> you have here to create a script for a cron job

this is unclear, what you're saying

**Reply by AgustinRamiroDiaz** [reply]
@odilitime I've edited it, I hope it's clear now

Paraphrasing and adding more details:

I want to take advantage of all the plugins that Eliza has to offer. I want to use Eliza just to run a script, without any AI agent, since my script is deterministic (run A, then B, then C).

My question is: how can I run Eliza without an AI agent? I want to use `Actions` and `Plugins`
